front/app/components/admin/ProjectableHeaderBgUploader/ProjectableHeaderBgUploader.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Have a look at my [PR with tests](https://github.com/CitizenLabDotCo/citizenlab/pull/3509/files). I'm using the best practices as advised by their docs and the react-testing library author. 
* prefer userEvent over fireEvent
* prefer ByRole over ByTestId

But my PR won't be merged before next week, so you'd have to duplicate some changes (e.g. the label in ImagesDropzone).

Maybe you can do a follow-up ticket to change these tests afterwards?

Reply:
Good to know. I'll fix it now.

Reply:
For e.g. image cropper, byTestId is fine. I tried and it's pretty hard. Only if it's easy to fix, use byRole.